src/main/java/org/killbill/billing/plugin/adyen/client/payment/service/AdyenCheckoutApiClient.java

```diff
@@ -27,13 +35,77 @@ public AdyenCheckoutApiClient(final AdyenConfigProperties adyenConfigProperties)
         checkoutApi = new Checkout(client);
```

Comment:
Do we set thread pool or connection configuration for Checkout client, if not then what is the default config and what is going to be the impact of using default config ?